README.md

```diff
@@ -169,11 +178,22 @@ export class AppComponent {
 }
 ```
 
+## Angular Universal
+
+For this to work with Angular Universal, you need to have a global window object defined in your `server.js` (the one in your Angular project derived from initiating an ssr project using Angular Universal) using any `window` npm lib like [window](https://www.npmjs.com/package/window) or [domino](https://www.npmjs.com/package/domino)
+
+Add the below to your `server.js`
+
+```js
+global['window'] = window;
```

Comment:
So you mean Angular Universal needs to install extra `window` package to make it to work ? Front end code will work fine right ?

Reply:
Yes. Angular Universal needs a window reference that can be installed explicitly. Frontend stays unaffected.